postgres/requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg2-binary
+s3fs
```

Comment:
why do we need those? 

it seems that in the `.py` files we mostly use `pandas` and `h3`

Reply:
see https://github.com/worldbank/DECAT_Space2Stats/pull/8/files#diff-3bd785b4bbbd346532389bb929cd18816234cc5607ce7c5ac9dc207c2cc0a66a for the full list of requirements. There are some dependencies used in the utility functions called in `api.py`.